test/test.js

```diff
@@ -129,6 +129,33 @@ describe('simplerdf', () => {
     assert(simple.isArray(posts))
   })
 
+  it('getter should use Arrays if there are multiple values', () => {
+    let graph = blogGraph.clone()
+
+    graph.add(rdf.createTriple(
+      rdf.createNamedNode(blogIri),
+      rdf.createNamedNode('http://schema.org/name'),
+      rdf.createLiteral('simple blog second name')))
+
+    let blog = simple(blogContext, blogIri, graph)
+
+    let names = blog.name
+
+    assert(simple.isArray(names))
```

Comment:
`name` is defined as `name: 'http://schema.org/name'`.
This means that this is not an array.. :S